src/components/Navbar/index.tsx

```diff
@@ -125,7 +126,7 @@ export interface NavbarProps extends React.HTMLAttributes<HTMLElement> {
  *
  * [View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Navbar/Navbar.stories.tsx)
  */
-export const Navbar = compose(setDefaultProps)(
+export const Navbar = flowRight(setDefaultProps)(
```

Comment:
As above I think this should be
```suggestion
export const Navbar = flow(setDefaultProps)(
```